client/components/loadable/index.js

```diff
@@ -9,11 +6,11 @@ import './style.scss';
  * @param {Object} props Component props.
  * @param {boolean} props.isLoading Flag used to display placeholder or content.
  * @param {string} props.display Defines how the placeholder is displayed: inline-block (default), inline or block.
- * @param {ReactNode} [props.placeholder] Custom placeholder content.
- * @param {ReactNode} [props.value] Content rendered when data are loaded. Has lower priority than `children`.
- * @param {ReactNode} [props.children] Content rendered when data are loaded. Has higher priority than `value`.
+ * @param {string} [props.placeholder] Custom placeholder content.
+ * @param {string} [props.value] Content rendered when data are loaded. Has lower priority than `children`.
+ * @param {string} [props.children] Content rendered when data are loaded. Has higher priority than `value`.
```

Comment:
I'm not sure changing `ReactNode` to `string` here is correct; the `Loadable` component should technically be able to receive a `string`, `number`, or any sort of component (`JSX.Element`, `ReactNode`, etc.); at least it does that in WCPay -- is that not the case here?

Did the linter complain about this? 🤔

Reply:
Yeah, the linter was yelling saying "`ReactNode` is not valid" - I was like "k, string it is then!" :D
Thanks for the suggestion, I now used `JSX.Element` and the waters seem to be calmer.